addons/mail/models/mail_thread.py

```diff
@@ -4531,6 +4531,21 @@ def _message_update_content(self, message, body, attachment_ids=None, partner_id
     # CONTROLLERS
     # ------------------------------------------------------
 
+    def _get_mail_thread_data_access_rights(self, store):
```

Comment:
This code belongs to controller, as it basically just format some dict / data. No need to add it on business models.